chia/cmds/wallet_funcs.py

```diff
@@ -482,23 +482,25 @@ async def print_balances(args: dict, wallet_client: WalletRpcClient, fingerprint
         else:
             print(f"Balances, fingerprint: {fingerprint}")
         for summary in summaries_response:
+            indent: str = "   "
             asset_id = summary["data"]
             wallet_id = summary["id"]
             balances = await wallet_client.get_wallet_balance(wallet_id)
             typ = WalletType(int(summary["type"]))
             address_prefix, scale = wallet_coin_unit(typ, address_prefix)
             total_balance: str = print_balance(balances["confirmed_wallet_balance"], scale, address_prefix)
-            print(f"{summary['name']}: {total_balance}")
-            print(f"   -Wallet ID: {wallet_id}")
-            print(f"   -Type: {typ.name}")
+            print()
+            print(f"{summary['name']}:")
+            print(f"{indent}{'-Wallet ID:'.ljust(23)} {wallet_id}")
```

Comment:
This works fine.  Note that there's also this.

```suggestion
            print(f"{indent}{'-Wallet ID:':23} {wallet_id}")
```